test/system/035-logs.bats

```diff
@@ -104,6 +104,8 @@ function _log_test_multi() {
     doit $cname1 "echo a1; echo a2"
     doit $cname2 "echo b1; echo b2"
 
+    run_podman wait $cname1 $cname2
```

Comment:
-f already wait for the containers to exit and is supposed to read until the end
I think https://github.com/containers/podman/issues/21914 is the much more likely problem with logs -f. The extra wait delay might fix that silently but I don't see this one here as proper fix.

Reply:
I didn't know that `-f` was waiting for containers to exit. I will remove this change.